pkg/apis/garden/v1beta1/types.go

```diff
@@ -760,6 +760,22 @@ type ShootStatus struct {
 	UID types.UID `json:"uid"`
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ShootBinding ties a shoot to a seed - used by the gardener-scheduler.
+// being used as a subresource for shoots: shoots/binding.
+// see core/v1/types for the official kubernetes Binding e.g used in the PodExpansion for Binding a Pod to a Node
+type ShootBinding struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	// More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#metadata
+	// +optional
+	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
+
+	// The target object that you want to bind to the standard object.
+	Target *Seed `json:"target" protobuf:"bytes,2,opt,name=target"`
```

Comment:
`omitempty`, optional tag missing

Reply:
Not optional here as the ApiServer will validate against the seeds existence

Reply:
Then it should be `Seed` instead of `*Seed`.